apps/org/models.py

```diff
@@ -1,3 +1,31 @@
+import random
+import string
+
+from django.conf import settings
 from django.db import models
+from django.utils.text import slugify
+
+
+class Organization(models.Model):
+    """An Organization."""
+    slug = models.SlugField(unique=True, max_length=255)
+    name = models.CharField(unique=True, max_length=255)
+    users = models.ManyToManyField(settings.AUTH_USER_MODEL, blank=True)
+
+    def __str__(self):
+        return self.name
+
+    def save(self, **kwargs):
+        """If we're saving an Organization for the first time, give it a slug based on name."""
+        if not self.id and not self.slug:
```

Comment:
Encapsulate this logic to make it more sustainable.
Either `self.set_slug()` or `set_slug(self)`. Either way is fine although it's great that the way this is written `set_slug(model)` would work with any model that has `id`, `name`. and `slug` fields. Both would also be an option, the first calling the second.